src/asmcnc/production/z_head_mechanics_jig/z_head_mechanics.py

```diff
@@ -0,0 +1,369 @@
+from kivy.uix.screenmanager import Screen
+from kivy.lang import Builder
+from kivy.clock import Clock
+
+from asmcnc.comms.yeti_grbl_protocol.c_defines import *
+from asmcnc.skavaUI import popup_info
+
+import matplotlib.pyplot as plt
```

Comment:
matplotlib MUST be included in ansible package install list, and this must be tested (you will need to uninstall and clear any cached install files in order to test)